datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -642,7 +642,7 @@ order by
 #[tokio::test]
 async fn test_physical_plan_display_indent() {
     // Hard code target_partitions as it appears in the RepartitionExec output
-    let config = SessionConfig::new().with_target_partitions(3);
+    let config = SessionConfig::new().with_target_partitions(9000);
```

Comment:
This was causing an amusing issue, as the mac runner happened to have 3 CPUs, it would replace the 3 with NUM_CPUS :facepalm:

Reply:
Here is another way it is handled ("explain normalizer"): 


https://github.com/tustvold/arrow-datafusion/blob/url-refactor/datafusion/core/tests/sql/mod.rs#L820-L858